File: src/C#/CreationalPatterns/Builder/Laptops/Director/LaptopStore.cs
using Laptops.Builder;
using System;
using System.Collections.Generic;
using System.Text;

namespace Laptops.Director
{
    public class LaptopStore
    {
        public void BuildLaptopConfiguration(LaptopBuilder laptopBuilder)
        {
            laptopBuilder.SetModel();
            laptopBuilder.SetCPUSeries();
            laptopBuilder.SetGPUModel();
            laptopBuilder.SetRAMType();
            laptopBuilder.SetRAMSize();
            laptopBuilder.SetDisplayType();
            laptopBuilder.SetSSDType();
            laptopBuilder.SetSSDCapacity();
            laptopBuilder.SetExtras();
        }
    }
}

File: src/JavaScript/CreationalPatterns/Builder/Laptops/Director/laptopStore.js

class LaptopStore {
    buildLaptopConfiguration(laptopBuilder) {
        laptopBuilder.setModel();
        laptopBuilder.setCPUSeries();
        laptopBuilder.setGPUModel();
        laptopBuilder.setRAMType();
        laptopBuilder.setRAMSize();
        laptopBuilder.setDisplayType();
        laptopBuilder.setSSDType();
        laptopBuilder.setSSDCapacity();
        laptopBuilder.setExtras();
    }
}

module.exports = LaptopStore;